### PR TITLE
Planefinding docs updates

### PR DESCRIPTION
**Pull request by keveleigh**

### CODE REVIEW

File: SpatialMapping/PlaneFinding/UnityAddon/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Plane Finding
+
+PlaneFinding add-on that can be used to find planar surfaces (ie: walls/floors/tables/etc) in mesh data, such as the mesh returned by spatial mapping.
```
Comment (davidkline-ms) [inline SpatialMapping/PlaneFinding/UnityAddon/README.md:3]:
```suggestion
PlaneFinding is an add-on that can be used to find planar surfaces (i.e.: walls/floors/tables/etc.) in mesh data, such as the mesh returned by spatial mapping.
```

File: SpatialMapping/PlaneFinding/UnityAddon/package.json [hunk 2]
```diff
@@ -1,8 +1,9 @@
 {
     "name": "com.microsoft.mixedreality.toolkit.planefinding",
     "displayName": "Mixed Reality Toolkit Plane Finding",
-    "version": "1.0.0-preview.1",
+    "version": "1.0.0",
```
Comment (davidkline-ms) [inline SpatialMapping/PlaneFinding/UnityAddon/package.json:4]:
```suggestion
    "version": "1.0.2",
```

To match the change log

**Review by davidkline-ms — CHANGES_REQUESTED** [review]
Couple minor suggestions

File: SpatialMapping/PlaneFinding/UnityAddon/package.json (hunk 2 above)
Comment (keveleigh) [reply SpatialMapping/PlaneFinding/UnityAddon/package.json:4]: Oh whoops, the changelog change you're looking at is actually for GPU Stats. Meant to break it out into a different PR, but it's in here...

**Review by davidkline-ms — APPROVED** [review]
:shipit:

**Pull request merged**